echo-pubsub-core/src/main/java/com/netflix/spinnaker/echo/pubsub/utils/NodeIdentity.java

```diff
@@ -73,7 +73,7 @@ private static String resolveHostname(String validationHost, int validationPort)
         try {
           socket = new Socket();
           socket.bind(new InetSocketAddress(address, 0));
-          socket.connect(new InetSocketAddress(validationHost, validationPort), 125);
+          socket.connect(new InetSocketAddress(validationHost, validationPort), 500);
```

Comment:
this looks good, just wonder if it would be useful to have this configurable.. but doesn't look like this is a bean... so i guess never mind